kaitai/kio.cpp

```diff
@@ -0,0 +1,216 @@
+#include <kaitai/kio.h>
+
+#include <iostream>
+#include <stdexcept>
+
```

Comment:
Probably warrants `using namespace kaitai;` rather than repeating it every time?

Reply:
Oh, I was just following the existing convention. What I'd really prefer is just wrapping the file in `namespace kaitai {}` but this doesn't work because of all of the local inclusions. But this works, done.